snomed/com.b2international.snowowl.snomed.datastore.tests/src/com/b2international/snowowl/snomed/core/ql/SnomedQueryLabelerRequestTest.java

```diff
@@ -202,5 +205,21 @@ public void bulkLabelExpressions() throws Exception {
 			Concepts.IS_A + " |" + isaPtUs + "|"
 		);
 	}
+	
+	@SuppressWarnings("unchecked")
+	@Test
+	public void wrongExpression() throws Exception {
+		String ptUk = "SNOMED CT Concept UK";
+		indexRevision(MAIN,
+				DocumentBuilders.concept(Concepts.ROOT_CONCEPT).preferredDescriptions(ImmutableList.of(
+						new SnomedDescriptionFragment(generateDescriptionId(), Concepts.SYNONYM, ptUk, Concepts.REFSET_LANGUAGE_TYPE_UK)))
+						.build());
+
+		try {
+			bulkLabel(List.of("test1", "test2", Concepts.ROOT_CONCEPT), SnomedConcept.Expand.PREFERRED_TERM, ImmutableList.of(ExtendedLocale.valueOf("en-x-" + Concepts.REFSET_LANGUAGE_TYPE_US)));
+		} catch (Exception e) {
```

Comment:
Either use JUnit 4's appropriate Rule implementation to catch and assert thrown exception type and message or use assertj's [exception assertion](https://assertj.github.io/doc/#assertj-core-exception-assertions).

Reply:
Done